backend/dal/Helpers/Extensions/PropertyExtensions.cs

```diff
@@ -95,8 +95,6 @@ private static IQueryable<Entity.Views.Property> GenerateCommonQuery(this PimsCo
                 query = query.Where(p => p.ProjectNumbers.Contains(filter.ProjectNumber));
             if (filter.IgnorePropertiesInProjects == true)
                 query = query.Where(p => p.ProjectNumbers == null || p.ProjectNumbers == "[]");
-            if (filter.InSurplusPropertyProgram == true)
```

Comment:
I think at one point this was the desired filter but it evolved over time and only got changed on the FE